### DISCUSSION TITLE
[Bug] Auto can't obtain GREAT on some sliders with Depth mod on

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#28612] [Q&A]
Take [Xaaaci. by Hey lululu](https://osu.ppy.sh/beatmapsets/999260#osu/2089944) as an example. With Depth mod only, Auto missed a few ticks while hitting the slider `00:56:060 (1)`. At 1:21 the cursor is completely out of the following circle.

![The slider Auto failed in Xaaaci.](https://github.com/ppy/osu/assets/110881926/6ae2dc9b-0748-493b-934e-b3b2377180aa)

When the OD, CS (and AR or not) is high, Auto would fail on some long or weirdly shaped sliders. It seems that though hit objects are moving "towards" us, the cursor won't exactly do like that, and we need to adjust the movement mechanism a bit for Depth.

**Comment by shiumano** [comment]
Auto is not compatible with mods that move the hit circle such as Wiggle and Repel, so I think it is incorrect to say that it is compatible with Depth.

………Wrong, Wiggle or so is fine. It's Magnetized that's included.

**Reply by Stoppedpuma** [reply]
Auto is compatible with those mods besides repel, automation mods just experience issues in lazer such as relax clicking early.

**Reply by FortDay-exe** [reply]
Auto doesnt seem to be compatible with Wiggle though. I created an Issue for this [#38708](https://github.com/ppy/osu/issues/38708)
